spec/models/group_spec.rb

```diff
@@ -242,6 +242,39 @@ def real_staff
       expect(GroupUser.where(user_id: staged.id).count).to eq(2)
     end
 
+    describe 'after updating automatic group members' do
+      fab!(:user) { Fabricate(:user) }
+
+      before { Jobs.run_immediately! }
```

Comment:
I don’t think this is necessary here since it is out of scope for the unit test. We just have to ensure the the job is enqueued with the right params.